ai-service/tests/conftest.py

```
@@ -1,12 +1,15 @@
 from typing import Generator
 import pytest
 import chromadb
+
+# ruff: noqa: E402
```

Ruff (0.12.2)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)